scss/7-utilities/_visibility.scss

```diff
@@ -32,50 +37,102 @@
   }
 }
 
-@each $name, $breakpoint in breakpoint-settings.$cads-breakpoints {
```

Comment:
I don't have strong feelings about this but out of curiosity - why did you decide to change the loop into separate classes?

Reply:
The rationale is in this commit: https://github.com/citizensadvice/design-system/pull/2372/commits/d84ca41b5521ea39ff9ecc985e18e41f296c0764

(Short version, it's to play nice with the implicit BEM linter rules so it can pick these classes up. One of the issues with generated CSS)